Automattic/jetpack
# [Boost] Fix bad options read

If the critical css option is emptied, it does not validate properly. That was causing an error - this PR fixes it by falling back to a default value instead. :) 

While I was at it, I added a unit test to cover the problem case.

## Proposed changes:
* Add a default value to the critical css option schema.

### Other information:

## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:
* Deactivate Boost.
* Delete all of your options.
* Reactivate Boost.
* Ensure the Critical CSS status object comes back valid, set to not_generated.
